azure-pipelines.yml

```diff
@@ -41,6 +41,10 @@ jobs:
       x86_64-msvc:
         TOOLCHAIN: stable-x86_64-pc-windows-msvc
         OTHER_TARGET: i686-pc-windows-msvc
+      x86_64-gnu:
+        # FIXME: switch to stable when 1.44 is released
+        TOOLCHAIN: nightly-x86_64-pc-windows-gnu
```

Comment:
I'd like to keep this on nightly permanently, so that we have *some* nightly testing for Windows (which enables a bunch of tests that are usually ignored).